test/Microsoft.NET.TestFramework/NuGetConfigWriter.cs

```diff
@@ -5,7 +5,6 @@ namespace Microsoft.NET.TestFramework
 {
     public static class NuGetConfigWriter
     {
-        public static readonly string DotnetCoreBlobFeed = "https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json";
         public static readonly string AspNetCoreDevFeed = "https://dotnet.myget.org/F/aspnetcore-dev/api/v3/index.json";
```

Comment:
Should we even be using myget feeds at this point, even if it's for testing? Unrelated to your change, but it caught my eye

Reply:
I'm not sure, I can make a separate pass over trying to get rid of it.